spirv_cross/src/wrapper.cpp

```diff
@@ -60,9 +60,42 @@ ScInternalResult sc_internal_compiler_hlsl_set_options(const ScInternalCompilerH
         } while (0);)
 }
 
-ScInternalResult sc_internal_compiler_msl_new(ScInternalCompilerMsl **compiler, const uint32_t *ir, size_t size)
+ScInternalResult sc_internal_compiler_msl_new(ScInternalCompilerMsl **compiler, const uint32_t *ir, size_t size,
+                                              const ScVertexAttributeOverride *const vat_overrides, size_t vat_override_count,
+                                              const ScResourceBindingOverride *const res_overrides, size_t res_override_count)
 {
-    INTERNAL_RESULT(*compiler = new spirv_cross::CompilerMSL(ir, size);)
+    INTERNAL_RESULT(do {
+            std::vector<spirv_cross::MSLVertexAttr> vertex_attributes;
```

Comment:
Looks good 👍 I think it would be nice to keep the `parse` signature consistent with HLSL and place the overrides into MSL `CompilerOptions` instead, but not a big deal either way.